wikipedia/track.json

```diff
@@ -0,0 +1,34 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Benchmark for search with Wikipedia data",
+  "search_applications": [
+    {}
+  ],
+  "indices": [
+    {
+      "name": "wikipedia",
+      "body": "wikipedia-{{index_mapping_type | default("minimal")}}-mapping.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "wikipedia",
+      "base-url": "https://rally-tracks.elastic.co/wikipedia",
+      "documents": [
+        {
+          "source-file": "documents.json.bz2",
```

Comment:
I've noticed that decompression with `pbzip2` only uses one cpu core. This leads to an unnecessary amount of time spent decompressing it on a powerful loaddriver.
Take a look at https://esrally.readthedocs.io/en/stable/adding_tracks.html?highlight=bzip2#sharing-your-track-with-others on how to compress it with `pbzip2` and thus reduce decompression time.

Reply:
@afoucret / @saikatsarkar056 I've uploaded a `pbzip2` compressed version of the corpora under the same bucket (it's called `rally-tracks/wikipedia/documents.json.pbzip2`), so feel free to swap those two (and update the compressed-bytes to `19064764394`).